README.md

```diff
@@ -258,9 +258,33 @@ _Analysis with serve mode_
 ```
 curl -X GET "http://localhost:8080/analyze?namespace=k8sgpt&explain=false"
 ```
+</details>
+
+## Additional AI providers  
+
+### Azure OpenAI  
+<em>Prerequisites:</em> an Azure OpenAI deployment is needed, please visit MS official [documentation](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal#create-a-resource) to create your own.
+
+To authenticate with k8sgpt, you will need the Azure OpenAI endpoint of your tenant `"https://your Azure OpenAI Endpoint"`, the api key to access your deployment, the deployment name of your model and the model name itself.  
+<details>
+
+### Run k8sgpt  
+To run k8sgpt, run `k8sgpt auth` with the `azureopenai` backend:  
+```
+k8sgpt auth --backend azureopenai --baseurl https://<your Azure OpenAI endpoint> --engine <deployment_name> --model <model_name>
+```
+Lastly, enter your Azure API key, after the prompt.
+
+Now you are ready to analyze with the azure openai backend:  
+```
+k8sgpt analyze --explain --backend azureopenai
+```
 
 </details>
 
+### Running local models
```

Comment:
This merge looks wrong, can you shift down the anonymisation header below the AI/models stuff?

Reply:
sorted !